src/modeler/loadFiles/CMakeLists.txt

```diff
@@ -21,12 +22,15 @@ target_include_directories(loadModelerFiles
 target_link_libraries(loadModelerFiles
         PUBLIC
         Antares::antares-study-system-model
+        Antares::linear-problem-api
         PRIVATE
         Antares::io
         Antares::yml-system
         Antares::yml-model
         Antares::model-converter
         Antares::modelerParameters
+        Antares::data-series-csv-importer
+        linear-problem-data-impl
```

Comment:
linear-problem-data-impl probably need an alias and be refered here as Antares::linear-problem-data-impl to be consistent with other libs